### PR TITLE
I added these attributes to the Span data type.

### PR DESCRIPTION
**Pull request by rhetoric101**
### Tell us why

This is a follow up to the docs work in doc-6617 that covered distributed tracing for Mobile. In this second PR related to that Jira, I created the following new attributes in the attribute dictionary on the Span event:

bytes.received
bytes.sent
connection.type
device.size
error.type
mobileApp.name
networkError
networkError.code
request.path
response.body

### Anything else you'd like to share?

If you want to see a spreadsheet of all the attributes for Mobile, check with rhetoric101, but the complete list overlaps with some existing attributes on the Span event type.

### CODE REVIEW

File: src/data-dictionary/events/Span/error.type.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: error.type
+type: attribute
+events:
+  - Span
+---
+
+Either HTTPError or NetworkFailure, depending on whether the error is a result of a failed request to a host or a failure on the cellular network.
```
Comment (urbiz-nr) [inline src/data-dictionary/events/Span/error.type.md:8]: Should HTTPError and Networkfailure be in code?

File: src/data-dictionary/events/Span/networkError.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: networkError
+type: attribute
+events:
+  - Span
+---
+
+The error message associated with the iOS NSURL Error code. See networkError.code for more information.
```
Comment (urbiz-nr) [inline src/data-dictionary/events/Span/networkError.md:8]: networkError in code?

**Review by urbiz-nr — COMMENTED** [review]
Left a couple of comments.

File: src/data-dictionary/events/Span/error.type.md (hunk 1 above)
Comment (rhetoric101) [reply src/data-dictionary/events/Span/error.type.md]: @urbiz-nr, I added the back ticks for code.

File: src/data-dictionary/events/Span/networkError.md (hunk 2 above)
Comment (rhetoric101) [reply src/data-dictionary/events/Span/networkError.md]: @urbiz-nr, I added the back ticks for code.

**Pull request merged**